ui/src/dropdown.tsx

```diff
@@ -109,24 +110,19 @@ const
         // HACK: Push clears args so run it after useEffect sets them due to model.value change.
         if (trigger) setTimeout(() => wave.push(), 0)
       },
-      selectAll = () => {
+      selectAll = (checked = true) => () => {
```

Comment:
Let's name this `select` (imperative) so that it's clear we should either select or deselect within the function. 

```suggestion
      selectAll = (select = true) => () => {
```

Reply:
Yes.

Reply:
Ideally, the intention should be to make the code better in all places rather than defaulting to suboptimal naming for the sake of keeping consistency. In this case, the change lies in variable naming only so a no-brainer.